docs/articles/blog/2020-7-15-introducing-the-testcafe-jenkins-plugin.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+title: "Introducing the TestCafe Jenkins plugin"
+permalink: /media/team-blog/:title.html
+isTeamBlog: true
+author: Pavel Redrukhin, Eugene Titerman
+---
+# Introducing the TestCafe Jenkins plugin
+
+TestCafe can capture videos and screenshots so you can debug your tests and examine page UI. You can record all the tests or only those that failed, generate multiple video files or stitch them together, and even single out specific interactions (see [the documentation](https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/screenshots-and-videos.html) for the full list of available options).
```

Comment:
What do you mean by "stitch them together"? Is a single video file generated for multiple tests? If so, can you phrase it this way instead?

Reply:
Would `You can record all the tests or only those that failed, generate unique video files for each test or record all of them in one take, and even single out specific interactions (see [the documentation](https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/screenshots-and-videos.html) for the full list of available options).` work?

Reply:
```suggestion
TestCafe can capture videos and screenshots so you can debug your tests and examine page UI. You can record all tests or only those that failed, generate unique video files for each test or record all of them in one take, and single out specific interactions. (For a full list of available options, see the following help topic: [Screenshots and Videos](https://devexpress.github.io/testcafe/documentation/guides/advanced-guides/screenshots-and-videos.html)).
```